recipe/build.sh

```diff
@@ -48,6 +48,9 @@ args+=" -D WITH_GZIP=ON"
 # plumed
 args+=" -D PKG_PLUMED=yes"
 args+=" -D PLUMED_MODE=runtime"
+# electrode
+args+=" -D PKG_ELECTRODE=yes"
+args+=" -D USE_INTERNAL_LINALG=yes"
```

Comment:
As we already have `blas` and `lapack` installed, I wonder if the `USE_INTERNAL_LINALG=yes` option is necessary. Did you try to disable this?